acceptance/tests/storeconfigs/deactivated_nodes.rb

```diff
@@ -84,11 +82,13 @@ class reactivated {
 
     step "deactivate the exporter node" do
       # Deactivate the node and wait until it's been processed
-      on database, "puppet node deactivate '#{exporter.node_name}'"
+      # FYI `puppet node deactivate` and `puppet node status` rely on the the
+      # pdb terminus, so they can only be run on the master
```

Comment:
True. We've never hit a problem since database == master on all our node tests :-/.